### PR TITLE
Fixed static main

### PR DESCRIPTION
**Pull request by BenergySD**
Fixed code in main causing compiler errors.

### CODE REVIEW

File: fix_this_ben.c [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #include <stdio.h>
 
-static int main(int argc, char** argv) {
-  printline("Hello, world.\n");
+int main(int argc, char** argv) {
+  printf("Hello, world.\n");
```
Comment (PaulJackson123) [inline fix_this_ben.c]: Please add a comment.

File: fix_this_ben.c [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #include <stdio.h>
 
-static int main(int argc, char** argv) {
```
Comment (PaulJackson123) [inline fix_this_ben.c:4]: Why did you remove `static`?

**Review by PaulJackson123 — CHANGES_REQUESTED** [review]
LGTM, but please address (PR) comments.

File: fix_this_ben.c (hunk 2 above)
Comment (BenergySD) [reply fix_this_ben.c:4]: C protocol

**Review by PaulJackson123 — APPROVED** [review]
LGTM
